common/jenkins-agents/nodejs12/docker/Dockerfile.ubi8

```diff
@@ -0,0 +1,61 @@
+FROM opendevstackorg/ods-jenkins-agent-base-ubi8:latest
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-agent-nodejs-12-rhel7-container" \
+      name="openshift4/jenkins-agent-nodejs-12-rhel7" \
+      architecture="x86_64" \
+      io.k8s.display-name="Jenkins Agent Nodejs" \
+      io.k8s.description="The jenkins agent nodejs image has the nodejs tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,agent,nodejs" \
+      maintainer="openshift-dev-services+jenkins@redhat.com"
```

Comment:
Are those lines still useful? If so, we might want to update them to reflect the fact rhel7 is no longer used here.

Reply:
Those are copied from the upstream image. They look outdated to me as well, however let's address labelling in another issue, and then make it consistent across all images.